client/src/Components/Pages/UploadImage/index.js

```diff
@@ -0,0 +1,173 @@
+import React, { Component } from "react";
+import axios from "axios";
+import Swal from "sweetalert2";
+
+import Select from "./../../Common/Select";
+
+import {
+  UploadImageWrapper,
+  ContentWrapper,
+  Heading,
+  SubHeading,
+  CardIcon,
+  ImageInput,
+  Paragraph,
+  Example,
+  Button,
+  Link,
+  SelectWrapper,
+  Error,
+  RightIcon
+} from "./UploadImage.style";
+import card from "./../../../assets/card.svg";
+import example from "./../../../assets/example.png";
+
+const label = "Trade";
+const placeholder = "Select your trade";
+
+export default class UploadImage extends Component {
+  state = {
+    tradeId: "",
+    trades: [],
+    error: ""
+  };
+
+  componentDidMount() {
+    axios.get("/api/trades").then(res => {
+      const { data } = res;
+      const trades = data.reduce((accu, current) => {
+        accu.push({ value: current._id, label: current.title });
+        return accu;
+      }, []);
+      this.setState({ trades });
+    });
+  }
+
+  handleChange = event => {
+    this.setState({ tradeId: event.target.value });
+  };
+
+  handleImageChange = event => {
+    const image = event.target.files && event.target.files[0];
+    var reader = new FileReader();
+
+    reader.onload = () => {
+      var dataURL = reader.result;
+      this.setState({
+        image: dataURL
+      });
+    };
+
+    this.setState(
+      {
+        imageFile: image
+      },
+      () => {
+        reader.readAsDataURL(image);
+      }
+    );
+  };
+
+  handleSubmit = event => {
+    event.preventDefault();
+    const form = new FormData();
+    if (!this.state.imageFile) {
+      this.setState({ error: "Please upload image" });
+    } else if (!this.state.tradeId) {
+      this.setState({ error: "Please select your trade" });
+    } else {
+      Swal.fire({
+        title: "Uploading!",
+        onBeforeOpen: () => {
+          Swal.showLoading();
+          this.setState({ error: "" });
+
+          form.append("verificationImage", this.state.imageFile);
+          form.append("tradeId", this.state.tradeId);
+
+          axios({
+            method: "post",
+            url: "/api/upload-verification-image",
```

Comment:
everythings looks fine but on my machine the post request goes to localhost:3000/api ... instead of localhost:8080... any idea why?

Reply:
do you mean in the network? or in terminal?